docs/prover/cli/options.md

```diff
@@ -251,19 +251,6 @@ Whenever you want to use a Solidity compiler executable with a non-default name.
 **Example**  
 `certoraRun Bank.sol --verify Bank:Bank.spec --solc solc8.1`
 
-### `--solc_args`
-
-**What does it do?**  
```

Comment:
I might leave this section in but replace it with
````
```{versionchanged} 4.0
Support for `--solc_args` has been removed; see {ref}`--optimize`, ...
```
````

Reply:
I am not sure it is a good idea. From what I see `--solc_args` was not used for long time by customers. It is only for solc options that are not supported explicitly  like --solc_allow_path, solc_optimize, solc_via_ir etc,  and it seems that what we provide gives and gave good coverage